docs/guides/create-webshop.md

```diff
@@ -0,0 +1,208 @@
+# Creating a webshop using OrchardCore.Commerce
+
+## Introduction
+
+Orchard Core's [Commerce](https://github.com/OrchardCMS/OrchardCore.Commerce) module can be used to create a fully functional webshop in a few simple steps. Check out the numbered steps to learn how to set up a webshop featuring highly customizable products and online payment, and read on to learn about using other key features such as order management, inventory management, promotions, and taxation.
+
+## Prerequisites
+- General familiarity with [Orchard Core](https://docs.orchardcore.net/en/latest) in general, see [Tutorials](https://docs.orchardcore.net/en/latest/docs/resources/tutorials/).
+- An installed Orchard Core website
+
+## Creating a basic webshop
+
+### Step 1 — Install the Commerce module using NuGet
+Get the latest version of the [Orchard Core Commerce NuGet](https://www.nuget.org/packages/OrchardCore.Commerce) package. Should you vie for further tips regarding NuGet usage with Orchard Core, see the [Getting Started](https://docs.orchardcore.net/en/latest/docs/getting-started) page.
+
+### Step 2 — Enable necessary features
+Certain Orchard Core Commerce features, as well as some prerequisite stock Orchard Core features, need to be enabled for the necessary functionality. This can be done under *Configuration > Features*. These features include the following:
+
+- **Orchard Core Commerce - Core**: Registers core components of Commerce features.
+- **Orchard Core Commerce - Payment** Provides the basics for online payment.
+- **Orchard Core Commerce - Payment - Stripe**: Provides online payment implementation using Stripe.
+- **Orchard Core Commerce - Session Cart Storage**: Provides shopping cart functionality.
+- **Widgets**: Allows rendering widgets in zones.
+- **Layers**: Allows rendering widgets across pages based on conditions.
+- **Feeds**: Adds feeds capabilities.
+- **Lists**: Adds a content type that can be used as a container.
+
+![Enable features in bulk.](../assets/images/create-webshop/step-2/enable-features.png)
+_Using the bulk enable option makes this process swift._
+
+Optionally, enable a few other useful Commerce features as well to get even more out of your webshop. More on these later in the [corresponding section](#other-useful-commerce-features), but these include:
+
+- **Orchard Core Commerce - Inventory**: Provides basic inventory management.
+- **Orchard Core Commerce - Promotion**: Allows specifying discounts and promotions.
+- **Orchard Core Commerce - Tax**: Provides tax-related functionality.
+- **Orchard Core Commerce - Custom Tax Rates**: Enables locally maintaining tax rates.
+
+### Step 3 — Create Shopping Cart Widget
+Shopping without a shopping cart is a difficult task, and while it's not outside the realm of possibility, let's make our lives easier by creating one.
```

Comment:
I'd say it's not that long, and this use case of the while clause works better (i.e. it more clearly refers back to the start of the sentence) this way. Could alternatively use a colon instead of the second comma as a middle ground, though the original version reads better to me:

> Shopping without a shopping cart is a difficult task: while it's not outside the realm of possibility, let's make our lives easier by creating one.

Reply:
It felt too long to me when reading it for the first time. You can't experience this as the author, because there was never a time when this text was new to you.
But if you want to keep it a single sentence, then please use a colon.

Reply:
The commas make it quite parsable, so it doesn't feel too long to me even in retrospect, but I do agree it's best to keep it on the simpler side. (I accidentally did it again there, damn it. I'm too used to writing this way.)